MapleServer2/Types/Player.cs

```diff
@@ -109,9 +109,9 @@ public class Player
         public int[] GroupChatId;
 
         // TODO: Rework to use Class Guild
-        public long GuildId;
-        public string GuildName;
-        public int GuildContribution;
+        public Guild Guild;
+        public GuildMember GuildMember;
+        public List<GuildApplication> GuildApplications = new List<GuildApplication>();
```

Comment:
Have you tested if this initialization will overwrite the return from the database?

Reply:
err?? I'm not sure. I've restarted the server a few times and havent had any overwrite happen. if thats what you mean

Reply:
I checked and it's ok!